main.tf

```diff
@@ -0,0 +1,138 @@
+# Create a VPC
```

Comment:
Please remove this comment. It's too obvious and don't provide any value

Reply:
did removed it